s3/replication/common/tests/system/test_put_bucket_replication.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+#
+# s3 get-bucket-replication api test script
+# Usage : python3 test_s3_get_bucket_replication.py
+#
+
+import asyncio
+import os
+import sys
+from config import Config
+import subprocess
+import fileinput
+
+def main():
+
+    config = Config()
+
+    # CONFIG OPTIONS
+    bucket_name = config.source_bucket_name
+    dest_bucket_name = config.target_bucket_name
+    iam_role = config.iam_role
+
+    subprocess.call(['cp', './tests/system/config/replication_policy_sample.json', 'temp_policy.json'])
```

Comment:
there are multiple Codacy warnings... is it normal?

Reply:
No Mehmet. One need to fix this.